### PR TITLE
Fixes #10086: Fix devel installer failing on RVM.

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: modules/katello_devel/templates/install_rvm.sh.erb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+USER=<%= @user %>
+
+SERVERS=(
+    hkp://keys.gnupg.net
+    hkp://pgp.mit.edu
+)
+
+for server in "${SERVERS[@]}"
+do
+    su -c "gpg --keyserver ${server} --recv-keys D39DC0E3" - $USER
```
Comment (ekohl) [inline modules/katello_devel/templates/install_rvm.sh.erb]: It might be nice if this somehow checked if it succeeded and if so stopped looping.

Comment (ehelms) [inline modules/katello_devel/templates/install_rvm.sh.erb]: Now you mention it :) I will look into this for the module.

Comment (ehelms) [inline modules/katello_devel/templates/install_rvm.sh.erb]: @ekohl opened an update here if you have time to look https://github.com/Katello/puppet-katello_devel/pull/37

**Comment by ehelms** [comment]
Updated with latest changes from devel module.

**Comment by stbenjam** [comment]
ACK

**Pull request merged**